.gitignore

```diff
@@ -52,3 +52,4 @@ out/
 etc/
 var/
 .idea/
+.DS_Store
```

Comment:
Revert this. You can solve this problem on your system by setting a global .gitignore, cf. for example http://egorsmirnov.me/2015/05/04/global-gitignore-file.html